### PR TITLE
ATS-913: Added tests for GIF and PNG

### PR DESCRIPTION
**Pull request by aymanthefirst**
Added GIF and PNG tests and their corresponding files for IPTC metadata extraction.

To see the specific steps taken, see the "How to" section in https://alfresco.atlassian.net/browse/ATS-913

### CODE REVIEW

**Review by KikoUK — APPROVED** [review]

File: alfresco-transform-tika/alfresco-transform-tika-boot/src/test/java/org/alfresco/transformer/TikaMetadataExtractsIT.java [hunk 1]
```diff
@@ -104,8 +104,8 @@ public void testTransformation(TestFileInfo testFileInfo)
                 testFile(MIMETYPE_IMAGE_JPEG, "jpg", "quickIPTC-EXT.jpg"),
                 testFile(MIMETYPE_IMAGE_JPEG, "jpg", "quickIPTC-multi-creator.jpg"),
                 testFile(MIMETYPE_IMAGE_JPEG, "jpg", "testJPEG_IPTC_EXT.jpg"),
-                testFile(MIMETYPE_IMAGE_GIF, "gif", "quick.gif"),
-                testFile(MIMETYPE_IMAGE_PNG, "png", "quick.png"),
+                testFile(MIMETYPE_IMAGE_GIF, "gif", "quickIPTC.gif"),
+                testFile(MIMETYPE_IMAGE_PNG, "png", "quickIPTC.png"),
```
Comment (PunchedChimera) [inline alfresco-transform-tika/alfresco-transform-tika-boot/src/test/java/org/alfresco/transformer/TikaMetadataExtractsIT.java:108]: These should be in addition to the original quick files, not instead.

**Review by PunchedChimera — CHANGES_REQUESTED** [review]

File: alfresco-transform-tika/alfresco-transform-tika-boot/src/test/java/org/alfresco/transformer/TikaMetadataExtractsIT.java (hunk 1 above)
Comment (aymanthefirst) [reply alfresco-transform-tika/alfresco-transform-tika-boot/src/test/java/org/alfresco/transformer/TikaMetadataExtractsIT.java:108]: I did not remove the original tests. See line [295 ](https://github.com/Alfresco/alfresco-transform-core/blob/ad767d21a24cc5575f8ed27589ebaee53d4afe85/alfresco-transform-tika/alfresco-transform-tika-boot/src/test/java/org/alfresco/transformer/TikaMetadataExtractsIT.java#L295)and [412](https://github.com/Alfresco/alfresco-transform-core/blob/ad767d21a24cc5575f8ed27589ebaee53d4afe85/alfresco-transform-tika/alfresco-transform-tika-boot/src/test/java/org/alfresco/transformer/TikaMetadataExtractsIT.java#L412)

Comment (PunchedChimera) [reply alfresco-transform-tika/alfresco-transform-tika-boot/src/test/java/org/alfresco/transformer/TikaMetadataExtractsIT.java:108]: Great thanks 👍  Looked a bit strange in the dif :)

**Review by PunchedChimera — APPROVED** [review]
Subject to green build 👍

**Pull request merged**